nixos/modules/services/matrix/matrix-sliding-sync.nix

```diff
@@ -1,10 +1,18 @@
 { config, lib, pkgs, ... }:
 
 let
-  cfg = config.services.matrix-synapse.sliding-sync;
+  cfg = config.services.matrix-sliding-sync;
 in
 {
-  options.services.matrix-synapse.sliding-sync = {
+  imports = [
+      (lib.mkRenamedOptionModuleWith {
+        sinceRelease = 2405;
```

Comment:
We don't do renames based on stateVersion and there is no reason to do so

Reply:
> We don't do renames based on stateVersion and there is no reason to do so

what would you recommend